flang/lib/Lower/IterationSpace.cpp

```diff
@@ -313,7 +315,7 @@ class IsEqualEvaluateExpr {
   static bool isEqual(const std::vector<A> &x, const std::vector<A> &y) {
     if (x.size() != y.size())
       return false;
-    const auto size = x.size();
+    const size_t size = x.size();
```

Comment:
I think the code uses both: `size_t` vs. `std::size_t`. We should pick one.

Reply:
Looks like `std::size_t` is more popular in the code base.